battle_test.go

```diff
@@ -22,11 +22,13 @@ func (a dumbAgent) Act(ctx *BattleContext) Turn {
 	panic("no opponents found")
 }
 
-type healAgent struct{}
+type itemAgent struct {
+	item int
+}
 
-// Always uses a potion on first Pokemon
-func (a healAgent) Act(ctx *BattleContext) Turn {
-	item := GetItem(ITEM_POTION)
+// Always uses the specified item on the first Pokemon
+func (a itemAgent) Act(ctx *BattleContext) Turn {
+	item := GetItem(a.item)
```

Comment:
Probably shouldn't change this, `healAgent` is used in other tests.

Use `RcAgent` instead